<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateUserDialog</class>
 <widget class="QDialog" name="CreateUserDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create User</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="firstNameLabel">
       <property name="text">
        <string>First name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="first_name_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lastNameLabel">
       <property name="text">
        <string>Last name:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="last_name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="fullNameLabel">
       <property name="text">
        <string>Full name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="initialsLabel">
       <property name="text">
        <string>Initials:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="initials_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="logonNameLabel">
       <property name="text">
        <string>Logon name:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="logonNamePreWindows2000Label">
       <property name="text">
        <string>Logon name (pre-Windows 2000):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="passwordLabel">
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="password_main_edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="passwordconfirmlabel">
       <property name="text">
        <string>Confirm password:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="password_confirm_edit">
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="upn_prefix_edit"/>
       </item>
       <item>
        <widget class="QComboBox" name="upn_suffix_edit"/>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="sam_name_domain_edit">
         <property name="enabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="sam_name_edit"/>
       </item>
      </layout>
     </item>
     <item row="8" column="1">
      <widget class="QCheckBox" name="show_password_check">
       <property name="text">
        <string>Show password</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Account options:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="must_change_pass_check">
        <property name="text">
         <string>User must change password on next logon</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cant_change_pass_check">
        <property name="text">
         <string>User cannot change password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="dont_expire_pass_check">
        <property name="text">
         <string>Don't expire password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="disabled_check">
        <property name="text">
         <string>Account disabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="button_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>first_name_edit</tabstop>
  <tabstop>last_name_edit</tabstop>
  <tabstop>name_edit</tabstop>
  <tabstop>initials_edit</tabstop>
  <tabstop>upn_prefix_edit</tabstop>
  <tabstop>upn_suffix_edit</tabstop>
  <tabstop>sam_name_edit</tabstop>
  <tabstop>sam_name_domain_edit</tabstop>
  <tabstop>password_main_edit</tabstop>
  <tabstop>password_confirm_edit</tabstop>
  <tabstop>show_password_check</tabstop>
  <tabstop>must_change_pass_check</tabstop>
  <tabstop>cant_change_pass_check</tabstop>
  <tabstop>dont_expire_pass_check</tabstop>
  <tabstop>disabled_check</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>button_box</sender>
   <signal>accepted()</signal>
   <receiver>CreateUserDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_box</sender>
   <signal>rejected()</signal>
   <receiver>CreateUserDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
